app/views/centers/index.html.slim

```diff
@@ -0,0 +1,8 @@
+h2 Existing List of PPIS Centers
+table border="1px" 
+  tbody
+    - @centers.each do |center|
+      tr
+        td#center_name= center.name
+        td#details= link_to 'Details', center_path(center)
```

Comment:
These ID's will make the DOMs invalid. (cannot have the duplicate IDs on the same HTML document)